contracts/MedianOracle.sol

```diff
@@ -182,7 +197,9 @@ contract MedianOracle is OwnableUpgradeable, IOracle {
             return (0, false);
         }
 
-        return (Select.computeMedian(validReports, size), true);
+        uint256 result = Select.computeMedian(validReports, size);
+        result = result.mulDiv(scalar, 10**18);
```

Comment:
This line enforces that `scalar` is in 18 decimals format, but that's not documented.

Let's update the `scalar` NatSpec to include this.
I'd be ok with also making an immutable `SCALAR_DECIMALS` property for the contract, which shouldn't use any extra storage.

Reply:
All numbers in this contract use the same 18 digit decimal representation so we can define DECIMALS=18, rather than SCALAR_DECIMALS=18 and DATA_DECIMALS=18